### ISSUE TITLE
Support using the same Redis server by multiple platform instances belonging to different tenants

### ISSUE THREAD

**Issue by tatarincev** [VirtoCommerce/vc-platform#1582]
**Problem:**
The current platform[ Scale Out architecture](https://virtocommerce.com/docs/vc2devguide/deployment/platform-scaleout) allows to use Redis server for local memory caches synchronization of multiple platform instances  that are related to one application belongs to one tenant.
So, the problem is to need to make possible to use the same Redis server for that purposes (cache sync)  for multiple platform instances that  belong to different tenants applications.
   
**Solution:**
Need to use a some unique logical identifier (group that can be defined in web.config) for all instances of the platforms that belongs to one tenant and use it on sending/reading  messages from a Redis server.
The code responds for communication logic with Redis server can be found by this link:
https://github.com/VirtoCommerce/vc-platform/blob/master/VirtoCommerce.Platform.Web/Cache/RedisCacheBackplane2.cs

**Comment by yecli** [comment]
We could use option 
```
<add key="VirtoCommerce:Cache:Redis:ChannelName" value="(Place name of desired channel here)" />
```
to specify channel name we want to use for caching.

**Comment by yecli** [comment]
Released in [v2.13.45](https://github.com/VirtoCommerce/vc-platform/releases/tag/v2.13.45).
